src/diff/diff_test.go

```diff
@@ -126,15 +134,11 @@ func TestDiffEqualConfig(t *testing.T) {
 		IsDefault:   false,
 	}
 	branchManifest := &manifest.BranchManifest{}
-	component := &model.Component{
-		ComponentKey: model.ComponentKey{
-			Id: "foo-bar",
-		},
-	}
+
 	configRemote := &model.Config{
 		ConfigKey: model.ConfigKey{
 			BranchId:    123,
-			ComponentId: "foo",
+			ComponentId: "foo-bar",
```

Comment:
Tu bol preklep v teste, ... kedze komponenta sa predtym zadavala ako objekt, tak sa tato chybna hodnota nikde nepouzila.

Teraz ked sa nacitava z cache, by to v tomto teste spadlo.